pyaedt/edb_core/edb_data/primitives_data.py

```diff
@@ -709,6 +709,22 @@ def width(self, value):
             else:
                 self.primitive_object.SetWidth(value)
 
+    @property
+    def length(self):
+        """Return the path length.
```

Comment:
```suggestion
        """Path length.
```
A description for a property shouldn't include a verb.